prediction_market_agent/agents/microchain_agent/functions.py

```diff
@@ -191,8 +191,9 @@ def description(self) -> str:
         return (
             f"Use this function to buy {self.outcome} outcome tokens of a "
             f"prediction market. The first parameter is the market id. The "
-            f"second parameter specifies how much {self.currency} you spend."
-            f"This is capped at {self.MAX_AMOUNT}{self.currency}."
+            f"second parameter specifies the VALUE of tokens to sell in "
+            f"{self.currency}. This is NOT the same as the number of tokens."
+            f"If this fails, try selling a smaller amount."
```

Comment:
Agent would always start buying new tokens. Now sells too.

Reply:
Does `sell` here mean `selling xDAI for outcome tokens`? If yes, this is correct, but I found it a bit confusing.
What do you think of rephrasing along the lines of "If fails, try purchasing a smaller amount of outcome tokens" or similar?
I think phrasing it in terms of `outcome tokens` is easier for the agent to understand - what do you think?

Reply:
My bad, I was copying this from another branch - it should be in the SellTokens description. Have moved! Hopefully it makes more sense in that context 😆 . I agree it's still confusing, but this can be removed when https://github.com/gnosis/prediction-market-agent-tooling/issues/195 is done

> I think phrasing it in terms of outcome tokens is easier for the agent

done